app/views/publish/courses/preview.html.erb

```diff
@@ -63,12 +63,12 @@
       <% if course.salaried? %>
         <li><%= govuk_link_to "Salary", "#section-salary" %></li>
       <% end %>
-      <li><%= govuk_link_to "Fees and financial support", "#section-fees-and-financial-support" %></li>
+      <li><%= govuk_link_to "Fees and financial support", "#section-financial-support" %></li>
```

Comment:
![image](https://user-images.githubusercontent.com/470137/182559788-f79937ae-6352-49c4-bf92-06da6b96f131.png)
vs
![image](https://user-images.githubusercontent.com/470137/182559973-f4ddca85-07f0-4674-b99d-60b8f94283f7.png)

It doesn't tally up, 
1. wording 
2. number of items
3. ordering

Reply:
Sorted